.golangci.yml

```diff
@@ -4,6 +4,7 @@ run:
   skip-dirs:
   - vendor
   - pkg/client/clientset/(.*)/fake
+  - pkg/client/injection
```

Comment:
Any reason to skip only `injection` :thinking:

Reply:
yes, so injection contains generated code container boilerplate for the reconciler. There were some lint errors I think that was causing the build tests to fail. It doesn't really make sense to fix generated code since it will be overwritten the next time we generate the code so I marked it as ignored from the linter.

Reply:
Right i totally agree with you.
One more point is along with `injection` i see most of the code from `pkg/client` contains generated code so is that okay if we skip all like `pipeline` https://github.com/tektoncd/pipeline/blob/master/.golangci.yml#L33 so that it gives clear information why we are skipping

Reply:
Good question: So pkg/client is mostly generated code but one folder is not: https://github.com/tektoncd/triggers/tree/master/pkg/client/dynamic/clientset

Reply:
ah i see the will leave the way you did :+1: